package.json

```diff
@@ -25,7 +31,6 @@
     "@types/prettier": "2.7.2",
     "eslint": "8.38.0",
     "eslint-config-upleveled": "4.0.12",
-    "stylelint": "15.4.0",
```

Comment:
stylelint is not necessary since we have no files to style or tests

Reply:
maybe it would be good to have a simple file with tests at `__tests__/index.module.css`, based on the content in the screenshot

makes it easier to update the screenshot in future

Reply:
[`b3d75e7` (#3)](https://github.com/upleveled/stylelint-config-upleveled/pull/3/commits/b3d75e7aab9a0cb91d91a95a2b54af41c1a7a31c)